service/src/tests/integration_tests.rs

```diff
@@ -1,126 +0,0 @@
-/*
-	Copyright 2021 Integritee AG and Supercomputing Systems AG
-
-	Licensed under the Apache License, Version 2.0 (the "License");
-	you may not use this file except in compliance with the License.
-	You may obtain a copy of the License at
-
-		http://www.apache.org/licenses/LICENSE-2.0
-
-	Unless required by applicable law or agreed to in writing, software
-	distributed under the License is distributed on an "AS IS" BASIS,
-	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-	See the License for the specific language governing permissions and
-	limitations under the License.
-
-*/
-
-use codec::Encode;
-use log::*;
-use sp_core::crypto::{AccountId32, Pair};
-use sp_keyring::AccountKeyring;
-use std::fs;
-use substrate_api_client::XtStatus;
-
-use itp_types::{CallWorkerFn, Request, ShieldFundsFn};
-use my_node_runtime::Header;
-use std::{thread::sleep, time::Duration};
-use substrate_api_client::{compose_extrinsic, UncheckedExtrinsicV4};
-
-use crate::tests::commons::*;
-use itp_api_client_extensions::TEEREX;
-use itp_enclave_api::{
-	enclave_base::EnclaveBase, remote_attestation::RemoteAttestation, sidechain::Sidechain,
-};
-use itp_settings::files::SIGNING_KEY_FILE;
-
```

Comment:
I'm interested to hear what @clangenb and @haerdib have to say about this. Since we were not running these integration tests anymore (because they don't work?), we remove them (since we have M6&M8 integration tests anyway)

Reply:
What's tested in these tests, is covered by M6 & M8. So we can remove them in terms of usability. Buuuut. With M6 & M8 we're only testing the happy flow. We should also be able to test some unhappy flows, from which some will need to be done from the untrusted side, I suppose.. and this file would give a little head start I guess.

But as they are,

Reply:
Yes, these were very old I, and test less that the M6/8 don't cover, so I told @gaudenzkessler that he could remove them instead of rewriting everything with the new `Arc<Enclave>`. They were written in my early days of rust, and we can rewrite integration tests from scratch much more idiomatic.

I reckon some of them were broken anyhow.